src/main/flight/pid.c

```diff
@@ -1100,9 +1100,11 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
         pidRuntime.oldSetpointCorrection[axis] = setpointCorrection;
 #endif
 
-        // Only enable feedforward for rate mode and if launch control is inactive
-        const float feedforwardGain = (flightModeFlags || launchControlActive) ? 0.0f : pidRuntime.pidCoefficient[axis].Kf;
+        // no feedforward in launch control
+        float feedforwardGain = (launchControlActive) ? 0.0f : pidRuntime.pidCoefficient[axis].Kf;
```

Comment:
You can also get rid of the braces around `launchControlActive`

Reply:
Thanks!  Didn't realise that was OK :-)